crates/bevy_winit/src/winit_windows.rs

```diff
@@ -72,6 +72,42 @@ impl WinitWindows {
             .with_decorations(window.decorations)
             .with_transparent(window.transparent);
 
+        #[cfg(all(
+            feature = "wayland",
+            any(
+                target_os = "linux",
+                target_os = "dragonfly",
+                target_os = "freebsd",
+                target_os = "netbsd",
+                target_os = "openbsd"
+            )
+        ))]
+        {
+            winit_window_builder = winit::platform::wayland::WindowBuilderExtWayland::with_name(
+                winit_window_builder,
+                window.app_id.clone(),
+                "",
+            );
+        }
+
+        #[cfg(all(
+            feature = "x11",
+            any(
+                target_os = "linux",
+                target_os = "dragonfly",
+                target_os = "freebsd",
+                target_os = "netbsd",
+                target_os = "openbsd"
+            )
+        ))]
+        {
+            winit_window_builder = winit::platform::x11::WindowBuilderExtX11::with_name(
+                winit_window_builder,
+                window.app_id.clone(),
+                "",
+            );
+        }
```

Comment:
```suggestion
        #[cfg(all(
            any(
                feature = "x11",
                feature = "wayland"
            ),
            any(
                target_os = "linux",
                target_os = "dragonfly",
                target_os = "freebsd",
                target_os = "netbsd",
                target_os = "openbsd"
            )
        ))]
        {
            winit_window_builder = winit_window_builder.with_name(
                window.app_id.clone(),
                "",
            );
        }
```

Reply:
Actually scratch this, can be cleaned up later when they stop requiring extension traits for calling this.

Reply:
Yeah I tried this before, then I found Rust doesn't allow this for now...